[FR] Update `Contests` (#5994)
ppy/osu-wiki: wiki/Contests/fr.md

# Concours

> Cette section est dédiée aux concours organisés par la communauté d'osu!

Reviewer: ```suggestion
Cette section est dédiée aux concours organisés par la communauté d'osu!.
```

## Concours officiels

Les concours officiels sont organisés (ou reconnus comme officiels) par l'[équipe d'osu!](/wiki/People/The_Team). La plupart des concours officiels ont leurs propres pages sur la [liste des concours](https://osu.ppy.sh/community/contests).

### Concours officiels de beatmapping

#### Monthly Beatmapping Contest

*Page principale : [Monthly Beatmapping Contest](/wiki/Contests/Monthly_Beatmapping_Contest)*

> Le **Monthly Beatmapping Contest**, dans son concept initial, était un concours de mapping où les mappeurs participants s'affrontaient pour créer la plus remarquable beatmap avec un [niveau de difficulté](/wiki/Beatmap/Difficulty#difficulty-levels) approprié à partir de Facile ou Normal en un mois.

Reviewer: ```suggestion
Le **Monthly Beatmapping Contest**, dans son concept initial, était un concours de mapping où les mappeurs participants s'affrontaient pour créer la plus remarquable beatmap avec un [niveau de difficulté](/wiki/Beatmap/Difficulty#difficulty-levels) approprié à partir de Easy ou Normal en un mois.
```

> Le **Monthly Beatmapping Contest**, dans son concept initial, était un concours de mapping où les mappeurs participants s'affrontaient pour créer la beatmap la plus remarquable avec plusieurs [niveaux de difficulté](/wiki/Beatmap/Difficulty#difficulty-levels) en commençant à partir de Easy ou Normal en un mois.

Reviewer 2: ```suggestion
Le **Monthly Beatmapping Contest**, dans son concept initial, était un concours de mapping où les mappeurs participants s'affrontaient pour créer la beatmap la plus remarquable avec plusieurs [niveaux de difficulté](/wiki/Beatmap/Difficulty#niveaux-de-difficulté) en commençant à partir de Easy ou Normal en un mois.
```

| [#12](https://osu.ppy.sh/home/news/2016-03-04-beatmapping-contest-12-osu) | IAHN - Candy Luv | ![osu!][osu!] | ![][flag_KR] [Taeyang](https://osu.ppy.sh/users/2732340) | ![][flag_CA] [Monstrata](https://osu.ppy.sh/users/2706438) | ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | [#1](https://osu.ppy.sh/home/news/2016-05-26-monthly-beatmapping-contest-12-results) |
| [#13](https://osu.ppy.sh/home/news/2016-12-09-monthly-beatmapping-contest-13-now-open) | BilliumMoto - HDHR | ![osu!][osu!] | ![][flag_MY] [RyoKazuka](https://osu.ppy.sh/users/6258586) | ![][flag_ID] [Fort](https://osu.ppy.sh/users/2805457) | ![][flag_DE] [Pho](https://osu.ppy.sh/users/3624692) | [#1](https://osu.ppy.sh/home/news/2017-04-02-monthly-beatmapping-contest-13-results) |

> Après une longue interruption, le Monthly Beatmapping Contest  a été relancé en janvier 2020 avec un tout nouveau calendrier, de nouvelles règles, une nouvelle procédure de vote et un nouveau système de récompense. De nombreux aspects du concours ont été radicalement modifiés afin de rendre le concours plus attrayant et plus intéressant, dont voici quelques exemples :

Reviewer: ```suggestion
Après une longue interruption, le Monthly Beatmapping Contest a été relancé en janvier 2020 avec un tout nouveau calendrier, de nouvelles règles, une nouvelle procédure de vote et un nouveau système de récompense. De nombreux aspects du concours ont été radicalement modifiés afin de rendre le concours plus attrayant et plus intéressant, dont voici quelques exemples :
```

| [#4](https://osu.ppy.sh/home/news/2018-08-19-aspire-2018-begins) | [Culprate - Acid Rain](https://osu.ppy.sh/beatmaps/artists/29) | ![][flag_TW] [SnowNiNo\_](https://osu.ppy.sh/users/2506267) | ![][flag_FI] [DTM9 Nowa](https://osu.ppy.sh/users/5428909) | ![][flag_LT] [seselis1](https://osu.ppy.sh/users/7321833) | [#1](https://osu.ppy.sh/home/news/2019-07-22-aspire-2018-results) |
| [#5](https://osu.ppy.sh/home/news/2020-04-09-aspire-v-a-new-format) | N'importe quelle musique disponible sur la [liste des Featured Artists](https://osu.ppy.sh/beatmaps/artists) jusqu'en avril 2020 | ![][flag_NL] [Mr HeliX](https://osu.ppy.sh/users/2330619) *(gagnant du choix de la communauté)*, ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699) *(gagnant du choix du mappeur)* | ![][flag_CA] [Meow Mix](https://osu.ppy.sh/users/3021634) | ![][flag_CN] [fanzhen0019](https://osu.ppy.sh/users/418699) | [#1](https://osu.ppy.sh/home/news/2020-08-23-aspire-v-results) |

#### osu! Mapping Olympiad

*Page principale : [osu! Mapping Olympiad](/wiki/Contests/Mapping_Olympiad)*

> L'**osu! Mapping Olympiad** est un concours de mapping qui se tient comme un successeur spirituel des anciens (avant 2020) Monthly Beatmapping Contests. Dans ce concours, les mappeurs participants sont chargés de créer soit une beatmap avec plusieurs [niveaux de difficulté](/wiki/Beatmap/Difficulty#difficulty-levels) en commençant à partir de Easy ou Normal, soit un certain nombre de difficultés pour n'importe quelle musique donnée dans chaque itération correspondante.

Reviewer 2: ```suggestion
L'**osu! Mapping Olympiad** est un concours de mapping qui se tient comme un successeur spirituel des anciens (avant 2020) Monthly Beatmapping Contests. Dans ce concours, les mappeurs participants sont chargés de créer soit une beatmap avec plusieurs [niveaux de difficulté](/wiki/Beatmap/Difficulty#niveaux-de-difficulté) en commençant à partir de Easy ou Normal, soit un certain nombre de difficultés pour n'importe quelle musique donnée dans chaque itération correspondante.
```

| [#7](https://osu.ppy.sh/home/news/2020-06-06-osu-mapping-olympiad-7-mania) | [Frums - Star Of The COME ON!!](https://osu.ppy.sh/beatmaps/artists/68), [IAHN - Welcome (Original Mix)](https://osu.ppy.sh/beatmaps/artists/3), [PUP - DVP](https://osu.ppy.sh/beatmaps/artists/79), [HyuN - ENCOUNTER](https://osu.ppy.sh/beatmaps/artists/33), et [Cranky - Ran](https://osu.ppy.sh/beatmaps/artists/23) | ![osu!mania][osu!mania] | ![][flag_US] [Toaph Daddy](https://osu.ppy.sh/users/7616811) et ![][flag_KR] [Kawawa](https://osu.ppy.sh/users/4647754) | *N/A* | ![][flag_PH] [lenpai](https://osu.ppy.sh/users/5314573) et ![][flag_CA] [BringoBrango](https://osu.ppy.sh/users/10274043) | [#1](https://osu.ppy.sh/home/news/2020-12-01-osu-mapping-olympiad-7-concludes) |

#### Community Mapping Contest

> Le **Community Mapping Contest** est un concours de mapping où les mappeurs participants sont libres de soumettre leur entrée pour n'importe quelle difficulté dans n'importe quel mode de jeu qu'ils désirent, après quoi toutes les entrées gagnantes, toutes difficultés et tous modes de jeu confondus, seront fusionnées en une seule beatmap.

Reviewer: ```suggestion
Le **Community Mapping Contest** est un concours de mapping où les mappeurs participants sont libres de soumettre leur entrée pour n'importe quelle difficulté dans n'importe quel mode de jeu, après quoi toutes les entrées gagnantes, toutes difficultés et tous modes de jeu confondus, seront fusionnées en une seule beatmap.
```

> | Numéro du concours | Musique | Mode de jeu | ![Couronne en or][GCrown] (Easy) | ![Couronne en or][GCrown] (Normal) | ![Couronne en or][GCrown] (Hard) | ![Couronne en or][GCrown] (Insane) | ![Couronne en or][GCrown] (Extra) | Results |

Reviewer: ```suggestion
| Numéro du concours | Musique | Mode de jeu | ![Couronne en or][GCrown] (Easy) | ![Couronne en or][GCrown] (Normal) | ![Couronne en or][GCrown] (Hard) | ![Couronne en or][GCrown] (Insane) | ![Couronne en or][GCrown] (Extra) | Résultats |
```

| [#1 (osu!mania (4K))](https://osu.ppy.sh/home/news/2016-09-28-community-mapping-contest-1-yuki-nadeshiko) | [yuki. - Nadeshiko Sensation](https://osu.ppy.sh/beatmaps/artists/4) | ![osu!mania][osu!mania] | ![][flag_CN] [Kayano](https://osu.ppy.sh/users/961752) | ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023) | ![][flag_ID] [LordRaika](https://osu.ppy.sh/users/3265023) | ![][flag_FR] [Ovoui](https://osu.ppy.sh/users/2282098) | ![][flag_ID] [DoNotMess](https://osu.ppy.sh/users/1596318) | [#1](https://osu.ppy.sh/beatmapsets/587388) |
| [#1 (osu!mania (7K))](https://osu.ppy.sh/home/news/2016-09-28-community-mapping-contest-1-yuki-nadeshiko) | [yuki. - Nadeshiko Sensation](https://osu.ppy.sh/beatmaps/artists/4) | ![osu!mania][osu!mania] | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | ![][flag_VN] [Kuo Kyoka](https://osu.ppy.sh/users/2596306) | [#1](https://osu.ppy.sh/beatmapsets/587388) |

#### Mapping With Rewards

> Le **Mapping With Rewards** est un concours de mapping dans lequel les mappeurs participants s'affrontent pour créer la meilleure beatmap (avec plusieurs [niveaux de difficulté](/wiki/Beatmap/Difficulty#difficulty-levels) en commençant à partir de Easy ou Normal) pour n'importe laquelle des musiques données dans chaque itération correspondante, sans délai strict.

Reviewer 2: ```suggestion
Le **Mapping With Rewards** est un concours de mapping dans lequel les mappeurs participants s'affrontent pour créer la meilleure beatmap (avec plusieurs [niveaux de difficulté](/wiki/Beatmap/Difficulty#niveaux-de-difficulté) en commençant à partir de Easy ou Normal) pour n'importe laquelle des musiques données dans chaque itération correspondante, sans délai strict.
```

> | Numéro du concours | Musique | Mode de jeu | ![Couronne en or](/wiki/shared/crown-gold.png "Première place") | ![Couronne d'argent](/wiki/shared/crown-silver.png "Seconde place") | ![Couronne de bronze](/wiki/shared/crown-bronze.png "Troisième place") | Results |

Reviewer: ```suggestion
| Numéro du concours | Musique | Mode de jeu | ![Couronne en or](/wiki/shared/crown-gold.png "Première place") | ![Couronne d'argent](/wiki/shared/crown-silver.png "Seconde place") | ![Couronne de bronze](/wiki/shared/crown-bronze.png "Troisième place") | Résultats |
```

#### Best of/Community Choice

Le **Best of/Community Choice** est un concours de vote public qui a lieu chaque année afin de déterminer les beatmaps les plus populaires aux yeux de la communauté d'osu!

> À son lancement, le concours a été nommé "Best of", car le principe initial et les attentes du concours étaient d'encourager les membres de la communauté à voter pour ce qu'ils considéraient comme "the best beatmaps of the year" ; cependant, comme la base de joueurs du jeu a continué à croître au fil du temps, le concours se transformait inévitablement de plus en plus en un concours de popularité chaque année, ce qui a incité le concours à être relancé comme "Community Choice" en 2017.

Reviewer: ```suggestion
À son lancement, le concours a été nommé "Best of", car le principe initial et les attentes du concours étaient d'encourager les membres de la communauté à voter pour ce qu'ils considéraient comme "the best beatmaps of the year" ; cependant, comme le nombre de joueurs a continué à croître au fil du temps, le concours se transformait inévitablement de plus en plus en un concours de popularité chaque année, ce qui a incité le concours à être relancé comme "Community Choice" en 2017.
```

| [#23 (Halloween 2020)](https://osu.ppy.sh/home/news/2020-09-18-halloween-2020-fanart-contest) | Halloween | ![][flag_BR] [CupcakeCrazier](https://osu.ppy.sh/users/12063010) | ![][flag_PH] [RianneRyan](https://osu.ppy.sh/users/17165886) | ![][flag_NZ] [SJPumpkin](https://osu.ppy.sh/users/4840900) | [#1](https://osu.ppy.sh/home/news/2020-10-11-halloween-fanart-contest-results) |
| [#24 (Winter 2020)](https://osu.ppy.sh/home/news/2020-09-11-winter-sports-2020-fanart-contest) | Sports d'hiver | ![][flag_US] [Tofumang](https://osu.ppy.sh/users/4817223) | ![][flag_ID] [CheekyIfa](https://osu.ppy.sh/users/14406940) | ![][flag_RU] [Dem4eg-](https://osu.ppy.sh/users/7206818) | [#1](https://osu.ppy.sh/home/news/2020-12-02-winter-2020-arrives) |
| [#25 (Spring 2021)](https://osu.ppy.sh/home/news/2021-04-23-spring-2021-fanart-contest) | Une journée dans le parc | ![][flag_US] [Tofumang](https://osu.ppy.sh/users/4817223) | ![][flag_MY] [Kyvrin](https://osu.ppy.sh/users/11589256) | ![][flag_BG] [risuuna](https://osu.ppy.sh/users/7266506) | [#1](https://osu.ppy.sh/home/news/2021-05-20-spring-fanart-contest-results) |

#### osu! Beatmap Blueprints Contest

> Le *osu! Beatmap Blueprints Contest* est un concours unique organisé pour commémorer le lancement de [Beatmap Blueprints](https://osu.ppy.sh/store/products/9) dans le [osu!store](https://osu.ppy.sh/store/listing). Dans ce concours, les utilisateurs participants se sont affrontés pour créer le meilleur plan d'une beatmap sur n'importe quel support visuel en utilisant toutes les ressources fournies dans le pack de marchandises Beatmap Blueprints.

Reviewer: ```suggestion
Le *osu! Beatmap Blueprints Contest* est un concours unique organisé pour commémorer le lancement de [Beatmap Blueprints](https://osu.ppy.sh/store/products/9) dans l'[osu!store](https://osu.ppy.sh/store/listing). Dans ce concours, les utilisateurs participants se sont affrontés pour créer le meilleur plan d'une beatmap sur n'importe quel support visuel en utilisant toutes les ressources fournies dans le pack de marchandises Beatmap Blueprints.
```

| Numéro du concours | ![Couronne en or][GCrown] | ![Couronne d'argent][SCrown] | ![Couronne de bronze][BCrown] | Résultats |
| :-: | :-- | :-- | :-- | :-- |
| [#1](https://osu.ppy.sh/home/news/2015-03-20-osu-beatmap-blueprints-available-contest) | *N/A* | *N/A* | *N/A* | *N/A* |

#### osu!idol

> **osu!idol** est un concours de chant où les participants de toute la communauté s'affrontent pour créer la meilleure reprise de musique (en solo ou avec un partenaire) à travers plusieurs étapes de jugement et de vote de la communauté. Le concours porte le nom de la franchise [Idols](https://fr.wikipedia.org/wiki/Idol_(franchise) "Wikipédia") et s'inspire principalement de divers concours de chant télévisés.

Reviewer: ```suggestion
L'**osu!idol** est un concours de chant dans lequel les participants de toute la communauté s'affrontent pour créer la meilleure reprise de musique (en solo ou avec un partenaire) à travers plusieurs étapes de jugement et de vote communautaire. Le concours porte le nom de la franchise [Idols](https://fr.wikipedia.org/wiki/Idol_(franchise) "Wikipédia") et s'inspire principalement de divers concours de chant télévisés.
```

| [#6 (2019)](https://osu.ppy.sh/home/news/2019-07-02-osu-idol-2019-auditions-now-open) | ![][flag_PH] [iamALieee](https://osu.ppy.sh/users/10093980) | ![][flag_US] [aceredshirt13](https://osu.ppy.sh/users/13736238) | ![][flag_US] [moof](https://osu.ppy.sh/users/8457202) | [iamALieee - White First Love](https://assets.ppy.sh/contests/83/entries/osu!idol%202019%20-%20iamALieee%20-%20White%20First%20Love.mp3) | [#1](https://osu.ppy.sh/home/news/2019-12-21-osu-idol-2019-finals-community-voting-results) |
| [#7 (2020)](https://osu.ppy.sh/home/news/2020-09-01-osu-idol-2020-auditions-now-open) | *N/A* | *N/A* | *N/A* | *N/A* | *N/A* |

#### osu!remix

**osu!remix** est un concours de remix de musique où les musiciens participants de toute la communauté sont chargés de créer le meilleur remix d'une musique choisie qui a été largement connue dans la communauté d'osu! d'une manière ou d'une autre. C'est le tout premier concours à être géré via la page [Liste des concours](https://osu.ppy.sh/community/contests).

> | Numéro du concours | Musique remixée | ![Couronne en or][GCrown] | ![Couronne d'argent][SCrown] | ![Couronne de bronze][BCrown] | Results |

Reviewer: ```suggestion
| Numéro du concours | Musique remixée | ![Couronne en or][GCrown] | ![Couronne d'argent][SCrown] | ![Couronne de bronze][BCrown] | Résultats |
```

| :-: | :-- | :-- | :-- | :-- | :-- |
| [#1](https://osu.ppy.sh/home/news/2016-05-06-osu-circles-remix-contest) | [nekodex - circles!](https://osu.ppy.sh/beatmaps/artists/1) | ![][flag_US] [BilliumMoto](https://osu.ppy.sh/users/3862471) | ![][flag_FR] [Kurokotei](https://osu.ppy.sh/users/398275) | ![][flag_NL] [happy30](https://osu.ppy.sh/users/27767) | [#1](https://osu.ppy.sh/home/news/2016-08-29-osuremix-contest-1-results) |

#### Skinning Contest

> The **Skinning Contest** is a skinning contest where artistic creators from all across the community compete to create the best skin in a limited time. 

Reviewer: ```suggestion
Le **Skinning Contest** est un concours de skinning dans lequel les créateurs artistiques de toute la communauté s'affrontent pour créer le meilleur skin en un temps limité. 
```

| Numéro du concours | ![Couronne en or][GCrown] | ![Couronne d'argent][SCrown] | ![Couronne de bronze][BCrown] | Résultats |
| :-: | :-- | :-- | :-- | :-- |
| [#1](/wiki/Contests/Skinning_Contest/1) | [FREEDOM DiVE↓](https://osu.ppy.sh/community/forums/topics/1293207) par ![][flag_US] [BTMC](https://osu.ppy.sh/users/3171691) | [Minimalist 2.0](https://osu.ppy.sh/community/forums/topics/1286223) par ![][flag_EE] [Krizto](https://osu.ppy.sh/users/13474556) | [notMiku](https://osu.ppy.sh/community/forums/topics/1292954) par ![][flag_RU] [NikStrim](https://osu.ppy.sh/users/10577559) | [#1](https://osu.ppy.sh/home/news/2021-05-27-skinning-contest-results) |

## Concours gérés par la communauté

Les *concours non officiels* sont des concours organisés par les membres de la communauté.

### osu! Beatmapping World Championship

> Le **osu! Beatmapping World Championship** est un concours de mapping par équipe où les équipes participantes, composées de mappeurs d'un même pays, d'une même région, voire d'un même continent (dans une certaine mesure), s'affrontent sur plusieurs musiques et étapes de mapping. Le concours est connu pour n'utiliser que des musiques disponibles dans la bibliothèque [Featured Artist](/wiki/Featured_Artists) pendant son déroulement.

Reviewer: ```suggestion
Le **osu! Beatmapping World Championship** est un concours de mapping par équipe dans lequel les équipes participantes, composées de mappeurs d'un même pays, d'une même région, voire d'un même continent (dans une certaine mesure), s'affrontent sur plusieurs musiques et étapes de mapping. Le concours est connu pour n'utiliser que des musiques disponibles dans la bibliothèque [Featured Artist](/wiki/Featured_Artists) pendant son déroulement.
```

| [#2 (2020)](/wiki/Contests/oBWC/2) | ![][flag_GB] Royaume-Uni ([hypercyte](https://osu.ppy.sh/users/9155377), [Aistre](https://osu.ppy.sh/users/4879380), [Altai](https://osu.ppy.sh/users/5745865), [Chugger](https://osu.ppy.sh/users/4491713), [DeviousPanda](https://osu.ppy.sh/users/4966334), [pocket-](https://osu.ppy.sh/users/6808091), et [Yusomi](https://osu.ppy.sh/users/4174940)) | ![][flag_AR] Argentine ([MaestroSplinter](https://osu.ppy.sh/users/6707918), [chucentry](https://osu.ppy.sh/users/2498731), [Lince Cosmico](https://osu.ppy.sh/users/6070370), et [Megafan](https://osu.ppy.sh/users/6632605)) | ![][flag_BR] Brésil ([Net0](https://osu.ppy.sh/users/5099768), [Enerugi](https://osu.ppy.sh/users/4475985), [Faito](https://osu.ppy.sh/users/9706291), [Kowari](https://osu.ppy.sh/users/5404892), [Maot](https://osu.ppy.sh/users/3914271), [Sakura Airi](https://osu.ppy.sh/users/8682057), et [Trynna](https://osu.ppy.sh/users/2652951)) | [#1](https://osu.ppy.sh/home/news/2020-10-22-beatmapping-world-championship-2020-concludes) |

### Pending Cup

> La **Pending Cup** est un concours de mapping organisé chaque année par des membres éminents de la communauté sinophone de mapping. musique et/ou catégories dans lesquelles les mappeurs de toute la communauté sont libres de concourir et de soumettre leurs entrées.

Reviewer: ```suggestion
La **Pending Cup** est un concours de mapping organisé chaque année par des membres éminents de la communauté sinophone de mapping. Chaque année, le concours présente jusqu'à trois musiques et/ou catégories dans lesquelles les mappeurs de toute la communauté sont libres de concourir et de soumettre leurs entrées.
```

| Catégorie | Mode de jeu | ![Couronne en or][GCrown] | ![Couronne d'argent][SCrown] | ![Couronne de bronze][BCrown] | Résultats |
| :-- | :-: | :-- | :-- | :-- | :-- |
| Featured Artist¹ | ![osu!][osu!] | ![][flag_KR] [Acylica](https://osu.ppy.sh/users/1943309) | ![][flag_ID] [AncuL](https://osu.ppy.sh/users/2449200) | ![][flag_CN] [yf\_bmp](https://osu.ppy.sh/users/1243669) | [#1](https://osu.ppy.sh/community/forums/topics/1198663) |
| Vocal² | ![osu!][osu!] | ![][flag_KR] [Heilia](https://osu.ppy.sh/users/9823042) | ![][flag_DE] [jamesjan3](https://osu.ppy.sh/users/6260705) | ![][flag_CN] [Garden](https://osu.ppy.sh/users/2849992) | [#1](https://osu.ppy.sh/community/forums/topics/1198663) |
| Instrumentale³ | ![osu!][osu!] | ![][flag_RU] [xbopost](https://osu.ppy.sh/users/6842421) | ![][flag_ID] [\[Keqing\]](https://osu.ppy.sh/users/8972308) | ![][flag_KR] [Acylica](https://osu.ppy.sh/users/1943309) | [#1](https://osu.ppy.sh/community/forums/topics/1198663) |

¹ La catégorie "Featured Artist" permet aux mappeurs de choisir entre deux chansons différentes à mapper : Zekk - Re\_construct et Shuu no Hazama \[Rainshower\].

> ² La catégorie  "Vocal" offre aux mappeurs le choix entre deux chansons différentes à mapper : nonoc - memento et DUSTCELL - DERO.

Reviewer: ```suggestion
² La catégorie "Vocal" offre aux mappeurs le choix entre deux chansons différentes à mapper : nonoc - memento et DUSTCELL - DERO.
```

³ La catégorie "Instrumentale" offre aux mappeurs le choix entre deux chansons différentes à mapper : onoken - Felys final remix et Nekomata Master - The Sky of Sadness.

### Newspaper Cup

> La **Newspaper Cup** est un concours de mapping similaire à la Pending Cup, mais qui met davantage l'accent sur les mappeurs débutants. Comme pour la Pending Cup, le concours est également organisé chaque année par des membres éminents de la communauté chinoise de mapping et est passé d'un concours particulièrement destiné aux membres de la communauté chinoise à un concours d'envergure mondiale.

Author: @Imakuri pour ici c'est pareil que pour [cette review](https://github.com/ppy/osu-wiki/pull/5994/files#r693333106) ?

Reviewer 3: Oui je pense que mettre "sinophone" serait plus censé en effet, c'est valable pour là aussi

> La **Newspaper Cup** est un concours de mapping similaire à la Pending Cup, mais qui met davantage l'accent sur les mappeurs débutants. Comme pour la Pending Cup, le concours est également organisé chaque année par des membres éminents de la communauté chinoise de mapping et est passé d'un concours particulièrement destiné aux membres de la communauté chinoise à un concours d'envergure mondiale.

Author: ```suggestion
La **Newspaper Cup** est un concours de mapping similaire à la Pending Cup, mais qui met davantage l'accent sur les mappeurs débutants. Comme pour la Pending Cup, le concours est également organisé chaque année par des membres éminents de la communauté sinophone de mapping et est passé d'un concours particulièrement destiné aux membres de la communauté sinophone à un concours d'envergure mondiale.
```

| Instrumentale² | ![osu!][osu!] | ![][flag_CA] [Kyumo](https://osu.ppy.sh/users/14689984) | ![][flag_US] [- ascended -](https://osu.ppy.sh/users/4564285) | ![][flag_CL] [Sunazuka Akira](https://osu.ppy.sh/users/8846632) | [#1](https://osu.ppy.sh/community/forums/topics/1274667) |

¹ La catégorie "Vocal" offre aux mappeurs un choix de cinq chansons différentes à mapper : Ariabl'eyeS - Unmei no Haguruma, Yunomi - Aimai Trip, KikuoHana - Desert Theater, Kaf - Tomadoi Telepathy, et Mafumafu - SNOBBISM.

² La catégorie "Instrumentale" offre aux mappeurs un choix de cinq chansons différentes à mapper : Umeboshi Chazuke - Dutch Courage!, The Flashbulb - Stretch Industry, MiddleIsland - Delrio, Nagakawa Koutarou - Hakata Tonkotsu Ramens, et roop - Chatblanc.

[flag_AE]: /wiki/shared/flag/AE.gif "Émirats arabes unis"
[flag_AR]: /wiki/shared/flag/AR.gif "Argentine"
[flag_AT]: /wiki/shared/flag/AT.gif "Autriche"
[flag_AU]: /wiki/shared/flag/AU.gif "Australie"
[flag_BE]: /wiki/shared/flag/BE.gif "Belgique"
[flag_BG]: /wiki/shared/flag/BG.gif "Bulgarie"
[flag_BR]: /wiki/shared/flag/BR.gif "Brésil"

> [flag_BY]: /wiki/shared/flag/BY.gif "Bélarus"

Reviewer 3: ```suggestion
[flag_BY]: /wiki/shared/flag/BY.gif "Biélorussie"
```
